deploy/overlay/kubernetes/patch-env.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jig-env
+  namespace: jig-workshop-worker
+  labels:
+    app: jig-workshop-worker
+    app.kubernetes.io/name: jig-workshop-worker
+    app.kubernetes.io/part-of: jig-workshop-worker
+data:
+  .env: |
+    APP_NAME=Jig
+    APP_ENV=k8s
+    APP_KEY=base64:aFOsf9UVdLX8bc9akpR9xEGIEx7+c4UShwPZ7XPrmCI=
```

Comment:
I need to remember to add a Security section to the README to go over the App Key...